tests/functional/pv/test_qos_limits.py

```
@@ -0,0 +1,568 @@
+        proj = project_factory()
+
+        pvc_obj = helpers.create_pvc(
+            sc_name=constants.DEFAULT_STORAGECLASS_RBD,
+            size="10Gi",
+            namespace=proj.namespace,
+            access_mode=constants.ACCESS_MODE_RWO,
+            volume_mode=constants.VOLUME_MODE_FILESYSTEM,
+        )
+        test_resources_cleanup["pvcs"].append(pvc_obj)
+
+        log.info(f"Waiting for PVC {pvc_obj.name} to transition to Bound phase...")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)